releasenotes/notes/uniform-superposition-gate-3bd95ffdf05ef18c.yaml

```
@@ -0,0 +1,23 @@
+---
+features:
+  - |
+    Implemented UniformSuperpositionGate class, which allows 

```

This allows Sphinx to link the text to the class docs
```suggestion
    Implemented :class:`.UniformSuperpositionGate` class, which allows 

```